lib/codeowners/cli/wizards/new_owner_wizard.rb

```diff
@@ -25,24 +28,55 @@ def suggest_fixing(line, new_owner)
 
         private
 
-        def prompt(line, new_owner)
-          ask(<<~QUESTION, limited_to: %w[y r i q])
-            Unknown owner: #{new_owner} for pattern: #{line.pattern}. Add owner to the OWNERS file?
-            (y) yes
-            (r) rename owner
-            (i) ignore owner in this session
-            (q) quit and save
+        def suggest_name_from_owners_list(new_owner)
+          require 'fuzzy_match'
+          search = FuzzyMatch.new(@owners_list.owners)
+          (suggested_owner, dice, _lev) = search.find_with_score(new_owner)
+          return suggested_owner if dice && dice > 0.6
+        end
+
+        def prompt(line, new_owner, suggested_owner)
+          prompt_options = build_prompt_options(suggested_owner)
+          accepted_options = filter_options(suggested_owner, prompt_options)
+          ask(<<~QUESTION, limited_to: accepted_options.keys)
+            #{print_prompt(line, new_owner, suggested_owner)}
+            #{print_options(accepted_options)}
           QUESTION
         end
 
+        def print_prompt(line, new_owner, suggested_owner)
+          prompt = "Unknown owner: #{new_owner} for pattern: #{line.pattern}."
+          prompt += " Did you mean #{suggested_owner}?" if suggested_owner
+          prompt + ' Add owner to the OWNERS file?'
```

Comment:
This second question is a bit confusing because you can only answer one question at a time. Can we skip this part? Because we're just going to add if we choose the Add option and it's explicit in the `(a) add a new owner` line.

Reply:
Done. Now the question is substitued according to the `suggested_owner` value:

```ruby
          if suggested_owner
            prompt + " Did you mean #{suggested_owner}?"
          else
            prompt + ' Add owner to the OWNERS file?'
          end
```